model/instance/lifecycle/patch.go

```diff
@@ -189,7 +189,9 @@ func Patch(i *instance.Instance, opts *Options) error {
 		}
 	}
 
-	managerUpdateSettings(i, clouderyChanges)
+	if opts.FromCloudery {
```

Comment:
shouldn't this logic be the other way?
if FromCloudery is true, we should not notify cloudery